# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/blindcoder/libmatroska/hotfixes.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile	2005-09-20 14:27:48.000000000 +0200
+++ ./Makefile	2005-09-20 14:28:06.000000000 +0200
@@ -103,7 +103,7 @@
 
 $(LIBRARY_SO): $(objects_so)
 	$(CXX) -shared -Wl,-soname,$(LIBRARY_SO_VER) -o $(LIBRARY_SO_VER) $(objects_so) -lebml
-	ln -s $(LIBRARY_SO_VER) $(LIBRARY_SO)
+	ln -fs $(LIBRARY_SO_VER) $(LIBRARY_SO)
 
 clean:	cleantest
 	rm -f $(objects) $(objects_so)
@@ -171,7 +171,7 @@
 install_sharedlib: $(LIBRARY_SO)
 	$(INSTALL) $(INSTALL_DIR_OPTS) -d $(libdir)
 	$(INSTALL) $(INSTALL_OPTS_LIB) $(LIBRARY_SO_VER) $(libdir)
-	ln -s $(LIBRARY_SO_VER) $(libdir)/$(LIBRARY_SO)
+	ln -fs $(LIBRARY_SO_VER) $(libdir)/$(LIBRARY_SO)
 
 ifneq ($(wildcard .depend),)
 include .depend
